### ISSUE TITLE
InceptionV3's Conv2d_2a_3x3.bn.running_var is incorrect

### ISSUE THREAD

**Issue by ypw-cortex** [Cadene/pretrained-models.pytorch#99]
https://gist.github.com/ypw-cortex/e924d2bcc689c6037821a8c21ac77cf9

The running_var of batchnorm should not be so small.

```py
bn = model.Conv2d_2a_3x3.bn
bn.running_var[11], bn.running_var[13]

(tensor(1.8647e-14), tensor(3.7400e-14))
```

The equivalent weight of batchnorm should not be so large.
```py
running_std = bn.running_var.sqrt() + bn.eps
eq_weight = bn.weight / running_std

tensor([  1.1262,   1.0825,   0.8164,   1.2763,   3.4662,   0.8985,   0.8461,
          1.2279,   0.3658,   2.4894,   0.8431, 999.8634,   0.3131, 999.8066,
          1.0949,   1.1219,   0.3374,   0.7440,   2.7890,   0.3605,   1.1327,
          0.5710,   0.5503,   0.6672,   0.6222,   0.3376,   0.4227,   0.9465,
          0.2870,   0.8486,   0.9414,   0.6925], grad_fn=<DivBackward1>)
```

The weight of the convolutional layer should not be so small too.
```py
conv = model.Conv2d_2a_3x3.conv
weight = conv.weight.data.mean(dim=-1).mean(dim=-1).mean(dim=-1)
weight[11], weight[13]

(tensor(2.9088e-10), tensor(2.5552e-10))
```

The other layer is the correct equivalent weight.
```py
bn = model.Conv2d_1a_3x3.bn
running_std = bn.running_var.sqrt() + bn.eps
weights = bn.weight / running_std

tensor([1.7271, 2.6709, 1.7531, 1.0139, 1.2894, 1.0954, 6.3732, 2.8702, 3.4773,
        1.6405, 2.6034, 1.4568, 1.0404, 3.3769, 2.1268, 2.3463, 1.2395, 1.1246,
        4.8683, 4.0129, 2.3488, 3.3749, 1.9351, 2.2495, 3.9881, 0.8731, 2.5312,
        1.1476, 2.0904, 2.0541, 2.6836, 5.9335], grad_fn=<DivBackward1>)
```

It can calculate the correct result in float32. But when I do 8-bit fixed-point, it can't get the correct result because of the great equivalent weight.

**Comment by ypw-cortex** [comment]
See https://github.com/pytorch/vision/issues/648
